externs/olx.js

```diff
@@ -4361,7 +4378,6 @@ olx.source.TileImageOptions.prototype.wrapX;
  *            cacheSize: (number|undefined),
  *            format: (ol.format.Feature|undefined),
  *            logo: (string|olx.LogoOptions|undefined),
- *            opaque: (boolean|undefined),
```

Comment:
Do you think it would make sense to mention the removal of `opaque` in the upgrade notes?